rust/processor/migrations/2023-09-11-164718_ut_remove_constraint/down.sql

```diff
@@ -0,0 +1,7 @@
+-- This file should undo anything in `up.sql`
+ALTER TABLE user_transactions DROP CONSTRAINT IF EXISTS fk_versions;
+ALTER TABLE signatures DROP CONSTRAINT IF EXISTS fk_transaction_versions;
+ALTER TABLE user_transactions
+ADD CONSTRAINT fk_versions FOREIGN KEY (version) REFERENCES transactions (version);
```

Comment:
When do we need to add a foreign key to a table?

Reply:
At this point never. This is only needed if you want to maintain consistency in a normal app (e.g. I want to make sure that when I'm writing to a post table that the user already exists).

Reply:
Given our work is really post processing...data cannot be wrong. That's a nice property of the blockchain.